src/components/VideoPlayerContexts/VideoPopoverMenuContext.js

```diff
@@ -16,6 +16,7 @@ function VideoPopoverMenuContextProvider({children}) {
     const {translate} = useLocalize();
     const [currentPlaybackSpeed, setCurrentPlaybackSpeed] = useState(CONST.VIDEO_PLAYER.PLAYBACK_SPEEDS[2]);
     const {isOffline} = useNetwork();
+    const isLocalFile = currentlyPlayingURL && (currentlyPlayingURL.startsWith('blob:') || currentlyPlayingURL.startsWith('file:'));
```

Comment:
Let's reuse the existing `ATTACHMENT_LOCAL_URL_PREFIX` const for consistency.

```suggestion
    const isLocalFile = currentlyPlayingURL && _.some(CONST.ATTACHMENT_LOCAL_URL_PREFIX, (prefix) => currentlyPlayingURL.startsWith(prefix));
```